packages/apps-config/src/api/spec/index.ts

```diff
@@ -139,6 +140,7 @@ const spec: Record<string, OverrideBundleDefinition> = {
   'jupiter-rococo': jupiterRococo,
   khala: phalaParachain,
   'kilt-parachain': kilt,
+  kylin,
```

Comment:
Just move it one down, see https://github.com/polkadot-js/apps/runs/2642112178?check_suite_focus=true#step:3:486

Reply:
Done